### PR TITLE
Update wit4java.py

### PR DESCRIPTION
**Pull request by Anthonysdu**

### CODE REVIEW

File: benchexec/tools/wit4java.py [hunk 1]
```diff
@@ -26,29 +26,21 @@ def name(self):
         return "wit4java"
 
     def cmdline(self, executable, options, task, rlimits):
-        return [executable] + options + list(task.input_files)
+        input_files =  list(task.input_files)
+        benchmark = input_files[-1]
+        packages = input_files[0:-1]
+        return [executable] + [benchmark] + ['--packages'] +  packages + options
```
[the remark this answers is not in the document]
Comment (Anthonysdu) [reply benchexec/tools/wit4java.py]: thanks, I removed this and added the support in the tool.

File: benchexec/tools/wit4java.py [hunk 2]
```diff
@@ -13,11 +13,11 @@ class Tool(benchexec.tools.template.BaseTool2):
 
     """
     Tool info for wit4java
-    (https://github.com/Anthonysdu/MSc-project/blob/main/jbmc/wit4java.py).
+    (https://github.com/wit4java/wit4java).
     """
 
     def executable(self, tool_locator):
-        return tool_locator.find_executable("wit4java.py")
+        return tool_locator.find_executable("wit4java")
```
[the remark this answers is not in the document]
Comment (Anthonysdu) [reply benchexec/tools/wit4java.py]: thanks! this is addressed.

**Comment by dbeyer** [comment]
@PhilippWendler Could you please have a look again?

File: benchexec/tools/wit4java.py [hunk 3]
```diff
@@ -30,25 +33,14 @@ def cmdline(self, executable, options, task, rlimits):
 
     def determine_result(self, run):
```
Comment (PhilippWendler) [inline benchexec/tools/wit4java.py:34]: I think that `determine_result` is not backwards compatible with the previous `wit4java.py`, right? Could you please add that as well? Either make the parsing generic enough that it looks for both old and new output, or set a field in `executable()` that stores the determined version and depend on that in the output parsing.

Also, `ToolNotFoundException` needs to be imported.

**Pull request merged**